### PR TITLE
Added ov_version parameter to the env tests

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added `ov_version` variable for the pytest in `test_examples`;
- Added `ov_version` variable for the pytest in the `test_install_*`;

### Reason for changes

- More flexible tests

### Related tickets

- 122860

### Tests

- Updated

### CODE REVIEW

File: tests/cross_fw/examples/conftest.py [hunk 1]
```diff
@@ -38,3 +41,8 @@ def backends_list(request):
 @pytest.fixture(scope="module")
 def is_check_performance(request):
     return request.config.getoption("--check_performance")
+
+
+@pytest.fixture(scope="module")
+def ov_module(request):
```
Comment (vshampor) [inline tests/cross_fw/examples/conftest.py:47]: `ov_version_override`

File: tests/cross_fw/examples/test_examples.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def example_test_cases():
 
 
 @pytest.mark.parametrize("example_name, example_params", example_test_cases())
-def test_examples(tmp_path, example_name, example_params, backends_list, is_check_performance):
+def test_examples(tmp_path, example_name, example_params, backends_list, is_check_performance, ov_module):
```
Comment (vshampor) [inline tests/cross_fw/examples/test_examples.py:45]: A good opportunity to set up type hints

**Review by vshampor — CHANGES_REQUESTED** [review]

**Review by alexsu52 — APPROVED** [review]
LGTM

**Comment by KodiaqQ** [comment]
run pylint pre-commit tests
run pytorch pre-commit tests
run tensorflow pre-commit tests
run openvino pre-commit tests
run onnx pre-commit tests

**Review by vshampor — APPROVED** [review]

**Pull request merged**